eng/build.ps1

```diff
@@ -16,10 +20,16 @@ foreach ($argument in $PSBoundParameters.Keys)
     "Release"    { $arguments += "/p:ConfigurationGroup=Release " }
     "ExtraArgs"  { $arguments += $ExtraArgs }
     "c"          { $arguments += "-rebuild " }
+    "Pack"       { $defaultargs = "-pack " }
+    "Restore"    { $defaultargs = "-restore " }
+    "Sign"       { $defaultargs = "-build /p:SignPackages=true " }
```

Comment:
eventually I think this should just become -sign but we will need to switch to arcade signing first.

Reply:
Agreed, this was my way around using Arcade's signing